astroquery/jplhorizons/core.py

```diff
@@ -445,6 +446,11 @@ def ephemerides_async(self, airmass_lessthan=99,
             Observer Table Quantities
             <https://ssd.jpl.nasa.gov/?horizons_doc#table_quantities>`_;
             default: all quantities
+        rts_flag: string, optional
+            Output data only at target rise/transit/set (RTS)
+            `RTS only print mode
+            <https://ssd.jpl.nasa.gov/?horizons_doc#rts>`_;
+            default: disabled
```

Comment:
In the docstring you could mention the accepted values for `rts_flag` (TVH, GEO, RAD) with a brief description to make the documentation as self-contained as possible.

Reply:
See my other comment